src/PrestaShopBundle/Event/ModuleManagementEvent.php

```diff
@@ -43,13 +43,21 @@ class ModuleManagementEvent extends Event
 
     private $module;
 
-    public function __construct(ModuleInterface $module)
+    private $systemClearCache;
+
+    public function __construct(ModuleInterface $module, bool $systemClearCache = true)
     {
         $this->module = $module;
+        $this->systemClearCache = $systemClearCache;
     }
 
     public function getModule()
     {
         return $this->module;
     }
+
+    public function getSystemClearCache()
+    {
+        return $this->systemClearCache;
```

Comment:
Is this something we could unit test 😄 ?

Reply:
I will check :)

Reply:
I think we don't have to uni test this class because it's like a dto

Reply:
My bad you're right 👍